packages/clay-toolbar/README.mdx

```diff
@@ -0,0 +1,75 @@
+---
+title: 'Toolbar'
+description: 'A toolbar is a set of actions related to a specific context that are grouped into a horizontal bar.'
+lexiconDefinition: 'https://liferay.design/lexicon/core-components/toolbars/'
+packageNpm: '@clayui/toolbar'
+sibling: 'docs/components/css-toolbar.html'
```

Comment:
Since this component isn't published yet, lets add "draft" so that its not published in the docs yet.